.scripts/travis_test.py

```diff
@@ -0,0 +1,151 @@
+"""A script that runs each of the Psi4NumPy reference implementations and checks that they exit successfully."""
+
+__author__ = "Daniel G. A. Smith"
+__license__ = "BSC-3Clause"
+__copyright__ = "(c) 2017, The Psi4NumPy Developers"
+
+import glob
+import subprocess
+import os
+import time
+import shutil
+import sys
+import numpy as np
+
+try:
+    import psi4
+except:
+    raise ImportError("The Python module 'psi4' was not found!")
+
+# How wide is the terminal
+ncolumns = shutil.get_terminal_size((80, 20)).columns
+
+# List of scripts to skip testing
+exceptions = ["Coupled-Cluster/TD-CCSD.py"]
+
+# List of folders to run the python scripts in
+reference_folders = [
+"Self-Consistent-Field",
+"Coupled-Cluster",
+"Moller-Plesset",
+"Symmetry-Adapted-Perturbation-Theory",
+"Electron-Propagator"]
+
+# List of folders to run the jupyter scripts in
+tutorial_folders = [
+"Tutorials/01_Psi4NumPy-Basics",
+"Tutorials/03_Hartree-Fock",
+"Tutorials/02_Linear_Algebra",
+# "Tutorials/04_Density_Functional_Theory",
+"Tutorials/05_Moller-Plesset",
+"Tutorials/06_Molecular_Properties"]
+
+### Helper functions
+
+def run_script(command):
+    """Runs a shell command and returns both the success flag and output"""
+    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
+    output = process.communicate()[0]
+    success = process.poll() == 0
+    return (success, output)
+
+def print_banner(title):
+    """Prints out a full banner across the entire screen"""
+
+    title = " " + title + " "
+
+    nequals = ncolumns - len(title)
+    nleft = nequals // 2
+
+    print(("=" * (nleft + nequals %2)) + title + ("=" * nleft))
+
+def print_flag(success):
+    if success:
+        print("\033[92m PASSED\033[00m")
+    else:
+        print("\033[91m FAILED\033[00m")
+
```

Comment:
Esc is itself a variable. Here's the file (https://github.com/psi4/psi4/blob/master/cmake/custom_color_messages.cmake)